### ISSUE TITLE
Add a "My Word List" tab in Ayamel Player

### ISSUE THREAD

**Issue by JKJensen** [BYU-ARCLITE/Ayamel-Examples#666]
Right now there are four tabs in the Ayamel Player: Definitions, Transcripts, and Annotations.

Please add a fourth tab called: "My Word List."

This will display [the word list](https://ayamel.byu.edu/words) that a user has accumulated so far.

**Comment by dro248** [comment]
@JKJensen, @verdude, @gliese1337, @DigitalProf 

A My Word List tab now exists on the Ayamel player. After meeting with Jeff, we both left with questions unanswered.

- What is the My Word List tab supposed to do?
- Should it only contain the words in text? Or should it contain the same functionality that the My Word List page has?
- Should the words be organized by language? Or should they all listed together?

**Comment by gliese1337** [comment]
I believe we have determined that it should contain the same functionality available on the My Word List page.
I would assume that words should be grouped by language.

**Comment by JKJensen** [comment]
The Ayamel Definitions tab is missing the "Add to my word list" button on each word.

**Comment by JKJensen** [comment]
I'd like to propose we remove the "My Word List" tab.

Now that we're streamlining using Ayamel, the best place for this functionality is where it currently is, under user settings. Making a redundant system for users is good, but probably not crucial at this time.

How close is this to being completed? If it's close, then continue. But if it hasn't been started yet, we can get rid of it. A blank "Word list" tab is live on the main site and users are wondering what it's supposed to do. Getting rid of it would be best for now.

**Comment by DigitalProf** [comment]
Indeed, this might make sense. A tab that does nothing raises questions. We could put it back when the functionality is working.

That said, I don't believe "Add to word list" is working. Let's test.
